components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_general.html

```diff
@@ -719,149 +718,20 @@ <h1 class="can-collapse defclose" data-name="details">
         
     <!-- ANNOTATIONS -->
 
+        <!-- templates are loaded based on config -->
+        {% for t in ome.metadata_pane|dictsort:"index" %}
```

Comment:
this is true if you will set, but for append it is easier to have that extra var, e.g. lets say you want to move `'{"name": "table", "label": "Tables", "index": "2"},'` for the very end. You only have to:

```
bin/omero config remove omero.web.ui.metadata_pane '{"name": "table", "label": "Tables", "index": "2"},'
bin/omero config append omero.web.ui.metadata_pane '{"name": "table", "label": "Tables", "index": "7"},'
```

we do not have `bin/omero append index `